fused-effects.cabal

```diff
@@ -51,10 +54,10 @@ library
                      , Control.Effect.Sum
                      , Control.Effect.Trace
                      , Control.Effect.Writer
-  build-depends:       base >=4.9 && <4.13
-                     , deepseq >=1.4.3 && <1.5
-                     , MonadRandom >=0.5 && <0.6
-                     , unliftio-core >= 0.1.2
+  build-depends:       base          ^>= 4.9
+                     , deepseq       ^>= 1.4.3
+                     , MonadRandom   ^>= 0.5
+                     , unliftio-core ^>= 0.1.2
```

Comment:
Using the PVP operator _should_ help us not need to do so much wrangling in future.

Reply:
We should switch to PVP operators when we move to 0.6.

Reply:
Mostly I want to find out if `stack` will deal.